### PR TITLE
Announcing Scala.js 0.6.14.

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

**Comment by sjrd** [comment]
Review by @gzm0 
Expected date of announcement: Wednesday 21.

File: _posts/news/2016-12-21-announcing-scalajs-0.6.14.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+{% highlight scala %}
+object Main {
+  @JSExportTopLevel("main")
+  def main(): Unit = { ... }
+}
+{% endhighlight %}
+
+which allows to call `main()` from JavaScript.
+`@JSExportTopLevel` methods must be declared in top-level `object`s.
```
Comment (gzm0) [inline _posts/news/2016-12-21-announcing-scalajs-0.6.14.md]: Are you deliberately saying top-level for simplicity here? `@JSExportTopLevel` is allowed in any static object.

File: doc/interoperability/export-to-javascript.md [hunk 2]
```diff
@@ -269,6 +269,27 @@ Not specifying `x` in this case will fail at runtime (since it does not have a d
 
 Just like `@JSExport`, `@JSExportNamed` takes the name of the exported method as an optional argument.
 
+## Exporting top-level methods
+
+While an `@JSExport`ed method inside an `@JSExport`ed object allows JavaScript code to call a "static" method,
+it does not feel like a top-level function from JavaScript's point of view.
+`@JSExportTopLevel` allows to export a method of a top-level object as a truly top-level function:
```
Comment (gzm0) [inline doc/interoperability/export-to-javascript.md]: Same comment about top-level here...

File: _posts/news/2016-12-21-announcing-scalajs-0.6.14.md (hunk 1 above)
Comment (sjrd) [inline _posts/news/2016-12-21-announcing-scalajs-0.6.14.md]: Yes, it's deliberate. People don't know what a "static" object is, so a full sentence would be necessary. The verbosity is not worth it IMO.

Comment (gzm0) [inline _posts/news/2016-12-21-announcing-scalajs-0.6.14.md]: I agree.

**Pull request merged**